diagnostic.md

```diff
@@ -11,21 +11,28 @@ Record your responses inside the fenced code blocks below each question.
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember generate component my-map
     ```
 
 1.  What files are created and/or edited to produce a component, and what are their responsibilities?
 
     ```md
-    <!-- your response here -->
+    In the `app/component` directory, `my-map/component.js` and
+    `my-map/template.hbs` are created.
+    It installs a component-test `components/my-map/component-test.js` inside
+    the tests/integration directory.
```

Comment:
Yep. This also asks for you to explain the responsibilities of the files.

Template files (`template.hbs`) are responsible for containing markup
(defining structure), binding data, and binding and dispatching actions.

Components objects (`component.js`) are responsible for defining actions,
declaring HTML attributes, and other presentation or behavior.

In general, templates are responsible for markup, while components are
responsible for presentation and behavior.

.5/1